src/domain/bitcoin/onchain/index.types.d.ts

```diff
@@ -79,22 +86,47 @@ type IncomingOnChainTxHandler = {
   ): { [key: WalletId]: BtcPaymentAmount } | ValidationError
 }
 
+type LookupOnChainPaymentArgs = {
+  txHash: OnChainTxHash
+  scanDepth: ScanDepth
+}
+
+type LookupOnChainReceiptArgs = LookupOnChainPaymentArgs
+
 interface IOnChainService {
   listActivePubkeys(): Pubkey[]
 
   getBalance(pubkey?: Pubkey): Promise<Satoshis | OnChainServiceError>
 
   getPendingBalance(pubkey?: Pubkey): Promise<Satoshis | OnChainServiceError>
 
-  listIncomingTransactions(
-    scanDepth: ScanDepth,
-  ): Promise<IncomingOnChainTransaction[] | OnChainServiceError>
+  listIncomingTransactions({
+    scanDepth,
+  }: ListIncomingTransactionsArgs): Promise<
+    IncomingOnChainTransaction[] | OnChainServiceError
+  >
+
+  listOutgoingTransactions({
```

Comment:
Same comment, why new export (didn't find it used anywhere outside package)

Reply:
this is part of the second PR, wil be used by cron